<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>665</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>191</width>
    <height>51</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="main" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>380</width>
     <height>580</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>580</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>380</width>
     <height>580</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#main {
background-color: rgb(255, 255, 255);
border-radius: 20px;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="top" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>160</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="styleSheet">
              <string notr="true">border: none;</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item alignment="Qt::AlignTop">
                 <widget class="QPushButton" name="hideButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#hideButton:hover{
padding-bottom:4px;
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="pc.qrc">
                    <normaloff>:/icons/icons/remove-sharp.png</normaloff>:/icons/icons/remove-sharp.png</iconset>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignTop">
                 <widget class="QPushButton" name="closeButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#closeButton:hover{
padding-bottom:4px;
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="pc.qrc">
                    <normaloff>:/icons/icons/close-sharp.png</normaloff>:/icons/icons/close-sharp.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item alignment="Qt::AlignHCenter|Qt::AlignTop">
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <pointsize>35</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>弹幕互动工具</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="inputContain" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item alignment="Qt::AlignHCenter">
                 <widget class="QLineEdit" name="username">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 1px solid gray;
            border-radius: 5px;
            padding-top: 14px;
            padding-left: 2px;
            padding-right: 2px;</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                  <property name="placeholderText">
                   <string>用户名</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QLineEdit" name="password">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 1px solid gray;
            border-radius: 5px;
            padding-top: 14px;
            padding-left: 2px;
            padding-right: 2px;</string>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                  <property name="placeholderText">
                   <string>密码</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QLabel" name="tips">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 7);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>354</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <property name="spacing">
            <number>0</number>
           </property>
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>记住密码</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QPushButton" name="loginButton">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#loginButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(50, 197, 203, 255), stop:1 rgba(57, 232, 156, 255));
            color: #f9ffff;
            font-size: 20px;
            font-weight: bold;
            border-radius: 5px;
}
#loginButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 214, 255, 255), stop:1 rgba(122, 217, 255, 255));
        }</string>
             </property>
             <property name="text">
              <string>登录</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>微信： o_x-Oncemore</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pc.qrc"/>
 </resources>
 <connections/>
</ui>
